main.py

```diff
@@ -10,6 +10,8 @@
 
 step_size = 20
 players_names = []
+# turtle starting position Y coordinates
+x = [0, -75, 75, -150, 150]

```

Comment:
Comments says `turtle starting position Y coordinates`, but the variable name is `x`. Why not `y`?